professor2-toolfile.spec

```diff
@@ -0,0 +1,29 @@
+### RPM external professor2-toolfile 1.0
+Requires: professor2
+%prep
+
+%build
+
+%install
+
+mkdir -p %i/etc/scram.d
+cat << \EOF_TOOLFILE >%i/etc/scram.d/professor2.xml
+<tool name="professor2" version="@TOOL_VERSION@">
+<client>
+<environment name="PROFESSOR2_BASE" default="@TOOL_ROOT@"/>
+</client>
+<use name="py2-numpy"/>
+<use name="py2-sympy"/>
```

Comment:
here and elsewhere, the py2-sympy tool won't be directly available - you should depend on py2-pippkgs instead.